internal/bcdb/provenance_query_processor.go

```diff
@@ -209,15 +241,15 @@ func (p *provenanceQueryProcessor) aclCheckForUserOperation(querierUserID, targe
 	return nil
 }
 
-func (p *provenanceQueryProcessor) aclCheckFor(querierUserID, targetUserID string) error {
+func (p *provenanceQueryProcessor) aclCheckForData(querierUserID string) error {
 	isAdmin, err := p.identityQuerier.HasAdministrationPrivilege(querierUserID)
 	if err != nil {
 		return err
 	}
 
-	if !isAdmin && querierUserID != targetUserID {
+	if !isAdmin {
 		return &errors.PermissionErr{
-			ErrMsg: "The querier is neither an admin nor requesting reads performed by the querier. Only an admin can query operations performed by other users.",
+			ErrMsg: "The querier [" + querierUserID + "] is not an admin. Only an admin can query historical data",
```

Comment:
make sure that handlers that consume this error return 403

Reply:
Done